product_pricelist_access/security/security.xml

```diff
@@ -16,4 +16,15 @@
             eval="[(4, ref('product_pricelist_access.restrict_product_pricelist'))]"
         />
     </record>
+      <record model="ir.rule" id="restrict_product_pricelist_item_record_rule">
```

Comment:
Not superimportant but please, note that this line is not aligned with the previous one.